common/remote/rmtssh.cpp

```diff
@@ -521,9 +523,22 @@ class CFRunSSH: public CInterface, implements IFRunSSH
                 if (res.length()==0)
                     PROGLOG("%d: %s(%d): [OK]",n+1,slaves.item(n),reply.item(n));
                 else if (strchr(res.str(),'\n')==NULL)
+                {
                     PROGLOG("%d: %s(%d): %s",n+1,slaves.item(n),reply.item(n),res.str());
+                    if (reply.item(n)){
+                        errCode = reply.item(n);
+                        errMsg.append(res.str());
+                    }
+                }
                 else
                     PROGLOG("%d: %s(%d):\n---------------------------\n%s\n===========================",n+1,slaves.item(n),reply.item(n),res.str());
+                    if (reply.item(n)){
+                        errCode = reply.item(n);
+                        errMsg.append(res.str());
+                    }
```

Comment:
The above 'if (reply.item(n))' block looks like it was supposed to be part of the else @ line 533....
it's indented, but block bracing is missing.